### PR TITLE
Disco commands

### LINKED ISSUE #333: Create CLI command to run DISCO analysis
- Prepare input files for DISCO - Add the input JSON file to send over to DISCO.
- Figure out directory structure on the CLI / Project Repo - Run within URBANopt scenario run directory. 
- Add command line call to run DISCO.
- #339

### PR DESCRIPTION
**Pull request by tanushree04**
### Pull Request Description

- Adds cli calls for disco

To test: 

- create disco project: uo create --disco -p `project`
This should create a disco folder that contains cost_database.xlsx, technical_catalogue.json, config.json.
This should include the electrical network.json feature file

- run disco project
- post process disco project
- run opendss
- run disco: uo disco -s `scenario file`  -f `feature file`
This should create a `disco` folder in the scenario folder in the run directory.
The folder should include `run_upgrade_cost_analysis.log` and 
- post process disco : uo process --disco -s `scenario file` -f `feature file`
This should create a run_upgrade_cost_analysis.log file, output_summary.json file, and a job_outputs folder

TODO: 
- update technical_catalog.json after error is fixed
- add test for disco post processing
- update scenario and reporting gem branches once released

### Checklist (Delete lines that don't apply)

- [x] Unit tests have been added or updated
- [x] All ci tests pass (green)
- [x] An [issue](https://github.com/urbanopt/urbanopt-cli/issues) has been created (which will be used for the changelog)
- [x] This branch is up-to-date with develop

### CODE REVIEW

File: spec/uo_cli_spec.rb [hunk 1]
```diff
@@ -295,6 +295,13 @@ def delete_directory_or_file(dir_or_file)
       expect(File.exist?(File.join(test_directory, 'run', 'two_building_ev_scenario', '2', 'finished.job'))).to be true
     end
 
+    it 'successfully installs python and dependencies' do
+      config =  '../example_files/python_deps/config.json'
+      FileUtils.rm_rf(config) if File.exist?(config)
+      system("#{call_cli} install_python")
+      expect(Dir.exist?(File.join('example_files','python_deps'))).to be true
```
Comment (kflemin) [inline spec/uo_cli_spec.rb:302]: can you expand this test a little bit to check for the config.json file too?

File: example_files/example_project.json [hunk 2]
```diff
@@ -122,15 +122,14 @@
         "id": "2",
         "name": "Restaurant 1",
         "type": "Building",
-        "building_type": "Food service",
+        "building_type": "Education",
```
Comment (kflemin) [inline example_files/example_project.json:125]: is this change needed?

File: example_files/example_project.json [hunk 3]
```diff
@@ -122,15 +122,14 @@
...
         "floor_area": 22313,
         "footprint_area": 22313,
         "number_of_stories": 1,
         "ev_charging": true,
         "ev_charging_behavior": "Business as Usual",
         "ev_percent": 100,
-        "ev_curtailment_frac": 0.5,
-        "ev_use_model_occupancy": true
+        "ev_curtailment_frac": 0.5
```
Comment (kflemin) [inline example_files/example_project.json:132]: this one too?

File: example_files/python_deps/install_python.sh [hunk 4]
```diff
@@ -33,6 +33,12 @@ function run_command
 	ret=$?
 	if [ $ret != 0 ]; then
 		error "command=[$@] failed return_code=$ret"
+		if ($FORCE_DOWNLOAD -eq 1) {
+			# This delay exists because we've observed cases where deleting the file fails because
+			# Windows says it is still in use. This seems to fix the issue.
+			Start-Sleep -Seconds 5
+			Remove-Item $path
+			}
```
Comment (tanushree04) [inline example_files/python_deps/install_python.sh:41]: @daniel-thom Thanks for the fix I added this here

**Comment by tanushree04** [comment]
@tareknrel I moved the read_data.py file here : https://github.com/urbanopt/urbanopt-cli/tree/disco_commands/example_files/scripts since we wanted to keep only the files we'll copy in an URBANopt project in the example_files>disco folder

**Review by kflemin — APPROVED** [review]
